src/lib/public-api.ts

```diff
@@ -10,6 +10,7 @@ export * from './components/table';
 export * from './components/alert';
 
 export { TerraComponentsModule } from './terra-components.module';
+export { TerraFilterModule } from './components/filter/filter.module';
```

Comment:
Hier sollte nicht nur das TerraFilterModule sondern alles was zur public-api des Filters gehört exportiert werden.
Wie schon zuvor vorgeschlagen, erstelle am besten eine `index.ts` im filter-Ordner und exportiere hier alles, was darin exportiert wird (siehe z.B. Alert)
```suggestion
export * from './components/filter';
```

Reply:
Ggf. musst du aber nochmal schauen ob wirklich alles was in der index.ts im Filter Ordner exportiert wird teil der public-api des Filters sein muss. Ansonsten würde ich das hier nochmal einschränken:
```suggestion
export { TerraFilterModule, FilterMenuDirective, FilterContainerDirective, ... } from './components/filter';
```

Reply:
im Prinzip kann alles was in der index einzeln gebraucht werden und damit auch wieder in der public API zur Verfügung gestellt werden! habe deshalb alles exportiert.